recirq/third_party/quaff/json_resolver.py

```diff
@@ -4,9 +4,9 @@
 
 
 def json_resolver(quaff_type: str) -> Optional[ObjectFactory]:
-    if quaff_type.startswith("quaff."):
+    if quaff_type.startswith("quaff"):
```

Comment:
The previous prefix `quaff.` seems more robust.
Is there some edge case that does not work with `quaff.` and needs `quaff` instead?

Reply:
No good reason, reverted.